CMakeLists.txt

```diff
@@ -43,8 +43,13 @@ set (Seastar_DPDK_MACHINE
   STRING
   "Configure DPDK for this processor architecture (if `Seastar_DPDK` is enabled). It configures -march or -mcpu")
 
+# Release versio used in shared library soname
+set (Seastar_VERSION "22.11.0")
```

Comment:
The version "22.11.0" was derived from its release date, as shown in commit 10b7d604d1f5037a733879d8d171d4405faebbe9. However, I recommend using `SEASTAR_API_LEVEL` instead of this date-based version as the SO (shared object) version number.

Here's why:

1. Applications select and link against shared libraries based on their SO versions
2. If we redistribute Seastar, we should use SO versions that meaningfully reflect API compatibility
3. The SO version should clearly indicate when breaking API changes occur - incrementing it signals that the new API is incompatible with previous versions

This approach provides better clarity around API compatibility compared to date-based versioning.

Reply:
- Used Seastar_API_LEVEL as soname.
- Version is still defined as the release date, can either:
    - revert to version being 1
    - define the version manually in CMakeLists.txt based on the last tag
    - create a script that will generate a VERSION file in the repository from the last tag made

Reply:
Have left VERSION in CMakeLists.txt as 1.  If needed, can update in a separate issue.